pkg/cmd/workspace/delete.go

```diff
@@ -97,17 +97,50 @@ var DeleteCmd = &cobra.Command{
 			return
 		}
 
-		res, err := apiClient.WorkspaceAPI.RemoveWorkspace(ctx, *workspace.Id).Execute()
-		if err != nil {
-			log.Fatal(apiclient.HandleErrorResponse(res, err))
+		if yesFlag {
+			res, err := apiClient.WorkspaceAPI.RemoveWorkspace(ctx, *workspace.Id).Execute()
```

Comment:
This block is a duplicate of the one below. Create a `removeWorkspace` function that both the blocks can call so the code is a bit more clean.